src/main/kotlin/no/nav/medlemskap/sykepenger/lytter/nais/SykepengerLytterRoutes.kt

```diff
@@ -54,6 +59,53 @@ fun Routing.sykepengerLytterRoutes(bomloService: BomloService) {
                 call.respond(HttpStatusCode.InternalServerError, t.message!!)
             }
         }
+        get("/brukerspørsmål") {
+            val callerPrincipal: JWTPrincipal = call.authentication.principal()!!
+            val azp = callerPrincipal.payload.getClaim("azp").asString()
+            secureLogger.info("EvalueringRoute: azp-claim i principal-token: {} ", azp)
+            val callId = call.callId ?: UUID.randomUUID().toString()
+            logger.info(
+                "kall autentisert, url : /brukerspørsmål",
+                kv("callId", callId)
+            )
+           val  requiredVariables:Map<String,String> = getRequiredVariables(call.request)
+            //kjøre ett kall mot lovme!
+             //må mappe til reauest objekt
+            //kalle lovme
+            //tolke svar
+            //svare med flex spesifikt respons
+            secureLogger.info ("Mocking data respons for request : ${requiredVariables["fnr"]} , ${requiredVariables["fom"]} ,${requiredVariables["tom"]} ")
+            val respons =FlexRespons(Svar.UAVKLART, Spørsmål.values().toSet())
+            call.respond(HttpStatusCode.OK,respons)
+        }
+    }
 
+
+}
+
+ fun getRequiredVariables(request: ApplicationRequest): Map<String, String> {
+    val headers = setOf("fnr")
+    val queryParams = setOf("fom","tom")
+    var returnMap = mutableMapOf<String,String>()
+    for (variabel in headers ){
+        try {
+            returnMap[variabel] = request.headers[variabel]!!
+        }
+        catch (e:NullPointerException){
+        throw BadRequestException("Header '$variabel' mangler")
+        }
     }
-}
\ No newline at end of file
+    for (variabel in queryParams ){
```

Comment:
```suggestion
    val queryParams = setOf("fom","tom") // Hard-coded value(s) we want to ensure are present
    for (variabel in queryParams ){
```

Flytt variabler til nærmest der de brukes, vi lever ikke i C lenger hvor alle variabler ligger på topp av funksjonen og må manuelt huskes på avh. hvilken linje i funksjonen man er =P